providers/fab/src/airflow/providers/fab/www/views.py

```diff
@@ -70,7 +70,7 @@ class FabIndexView(IndexView):
     def index(self):
         if g.user is not None and g.user.is_authenticated:
             token = get_auth_manager().get_jwt_token(g.user)
-            return redirect(f"/webapp?token={token}", code=302)
+            return redirect(f"{conf.get('fastapi', 'base_url')}/?token={token}", code=302)
```

Comment:
(heads up @vincbeck - this doesn't seem to be tested. not going to worry about it in this PR though)